component/exporter.go

```diff
@@ -77,6 +77,7 @@ type ExporterFactory interface {
 		ctx context.Context,
 		params ExporterCreateParams,
 		cfg config.Exporter,
+		nextConsumer consumer.Traces,
```

Comment:
Why are these additional parameters being added to the methods on this interface?

Reply:
These methods did not have any parameter to differentiate between a TracesExporter, MetricsExporter or a LogsExporter. The ProcessorFactory interface methods had this additional parameter, so I picked it up from there.

Reply:
Exporter is the last component in a pipeline so does not have a "next" component.